### PR TITLE
Yeoman article fixes

### PR DESCRIPTION
**Pull request by spboyer**
These edits set the yeoman generator to the specific version that support the article walkthrough.  The generators have been updated for csproj. Therefore installing the latest version i.e. 
`npm install -g generator-aspnet` would have version `0.3.1` installed and developers confused as to why the walkthroughs don't work.

They must install the 0.2.6 version: `npm install -g generator-aspnet@0.2.6`.

Until the updates for the tutorials are complete, these edits fix that.

### CODE REVIEW

**Comment by spboyer** [comment]
@tdykstra @RickAndMSFT

File: aspnetcore/client-side/yeoman.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: Building projects with Yeoman | Microsoft Docs
 author: ardalis
-description: 
-keywords: ASP.NET Core,
+description: This article walks through building an ASP.NET Core web application using the Yeoman generator on macOS.
+keywords: ASP.NET Core, Yeoman
 ms.author: riande
```
Comment (tdykstra) [inline aspnetcore/client-side/yeoman.md]: Consider changing this to your alias

File: aspnetcore/security/key-vault-configuration/sample/README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-    - `MySecret`: `secret_value_1`
-    - `Section--MySecret`: `secret_value_2`
-  * Register the application with Azure Active Directory
-  * Authorize the application to use the secrets
-2. Provide the configuration data to the `appsettings.json` file of the sample:
-  * `Vault`: Azure Key Vault Name
-  * `ClientId`: Azure AD Application Id
-  * `ClientSecret`: Azure AD Application Key
-3. Restore, build, and run the application
+# Key Vault Configuration Provider sample application
```
Comment (tdykstra) [inline aspnetcore/security/key-vault-configuration/sample/README.md]: Why does Git think that every line in this file changed though it doesn't look like they changed at all?

Comment (spboyer) [inline aspnetcore/security/key-vault-configuration/sample/README.md]: yes thats weird

File: aspnetcore/client-side/yeoman.md (hunk 1 above)
Comment (spboyer) [inline aspnetcore/client-side/yeoman.md]: i did.

**Pull request merged**